outputs.tf

```diff
@@ -8,11 +8,6 @@ output "server_name" {
   value       = azurerm_mysql_server.mysql.name
 }
 
-output "database_name" {
-  description = "The database name of MySQL Server."
-  value       = azurerm_mysql_database.mysql.name
-}
```

Comment:
why take this out?

Reply:
Because I'm using a map now to create multiple databases, thus the dbs var.

Reply:
ohh yeah that'll mess up the output that I don't know if we need. I think that makes this a major release?

Reply:
Yeah, I was tagging this 3.0.0